testing/kokoro/encrypt-secrets.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright 2021 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+TESTING_ROOT=$( dirname "$DIR" )
+
+cd $TESTING_ROOT
+
+PROJECT_ID="golang-samples-tests"
```

Comment:
Maybe only set this if it's not provided? I'm thinking about people who want to run this on their own infra.